examples/connectivity/plot_adhd_spheres.py

```diff
@@ -0,0 +1,69 @@
+"""
+Extracting brain signal from spheres
+====================================
+
+This example extract brain signals from spheres described by the coordinates
+of their center in MNI space and a given radius in millimeters. In particular,
+this example extracts signals from Default Mode Network regions and compute a
```

Comment:
Well, redundancy is on purpose and I don't understand why we shouldn't say what we are doing in the example. Can we have another opinion?